### PR TITLE
Do not fail when wc resource has an unknown attribute

### PR DESCRIPTION
**Pull request by pmdeazeta**

### CODE REVIEW

File: lib/woocommerce_api/resource.rb [hunk 1]
```diff
@@ -32,6 +32,14 @@ class Resource
     include WoocommerceAPI::AttributeAssignment
     self.include_root_in_json = true
 
+    def initialize(params={})
+      params.each do |attr, value|
+        self.public_send("#{attr}=", value) if self.respond_to? attr
+      end if params
+
+      super()
+    end
```
Comment (rhearnorth) [inline lib/woocommerce_api/resource.rb]: if we do this, we don't need super anymore. right?

Comment (rhearnorth) [inline lib/woocommerce_api/resource.rb]: also, don't need "if param"

**Comment by rhearnorth** [comment]
LGTM

**Comment by bradleypriest** [comment]
If we included Virtus.model after ActiveModel::Model, this would junst work, not sure what would break though

**Comment by rhearnorth** [comment]
@bradleypriest Nope, I tried

**Pull request merged**